defaults/defaults.go

```diff
@@ -19,8 +19,7 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector/config"
 	"github.com/open-telemetry/opentelemetry-collector/exporter"
 	"github.com/open-telemetry/opentelemetry-collector/exporter/fileexporter"
-	"github.com/open-telemetry/opentelemetry-collector/exporter/jaeger/jaegergrpcexporter"
-	"github.com/open-telemetry/opentelemetry-collector/exporter/jaeger/jaegerthrifthttpexporter"
+	"github.com/open-telemetry/opentelemetry-collector/exporter/jaegerexporter"
```

Comment:
This is an exported packages and we are breaking other people's code by doing all these changes. I am very reluctant about this. I believe it is best to avoid renaming jaegergrpcexporter.

Reply:
Ok can revert the package - but given that the collector is pre 1.x, wouldn't now be the best time to fix these inconsistencies? Especially if the exporter type is going to result in a breaking change anyway?

Reply:
Yes, now is a better time than later.

I have just checked the contrib code, it appears we are not directly importing this package from that repo, so most likely others don't either. This means it is very unlikely we break anyone's code.

I am OK with renaming, but we will need to announce in gitter before merging this PR. @open-telemetry/collector-approvers are you OK?